libs/types.ts

```diff
@@ -169,12 +169,10 @@ export interface Project {
   project_tags: string[]
 }
 
-export type Projects = Project[]
```

Comment:
To me it looks unclear which type is under `projects` could be an Array, a Map, ...
I find it easier to keep simple types such as `Project` and then apply collection type

Reply:
ok